web/src/features/map/Map.tsx

```diff
@@ -198,6 +201,23 @@ export default function MapPage(): ReactElement {
     // TODO: Remove loading overlay
     // TODO: Show error message to user
   };
+
+  const onDragStart = () => {
+    setIsMoving(true);
+  };
+
+  const onDragEnd = () => {
+    setIsMoving(false);
+  };
+
+  const onZoomStart = () => {
+    setIsMoving(true);
+  };
+
+  const onZoomEnd = () => {
+    setIsMoving(false);
+  };
```

Comment:
While I realize why you did it this way and it's super clear I don't think we should duplicate exact functionality. Perhaps combining them like the following suggestion would work?
 ```suggestion
  const onDragOrZoomStart = () => {
    setIsMoving(true);
  };

  const onDragOrZoomEnd = () => {
    setIsMoving(false);
  };
```
Naming is just a suggestion, feel free to change it to something else if you want to go forward with this.

Reply:
Sure